smalldata_tools/SmallDataUtils.py

```diff
@@ -321,6 +321,15 @@ def getUserData(det):
     for key in det_dict:
         if key.find('ragged')>=0:
             newDict['ragged_%s'%(key.replace('ragged_',''))] = det_dict[key]
+        elif key.find('var')>=0:
+            fname = key.split('_')[-1]
+            dname = key.replace('_%s'%fname,'').replace('_var','')
```

Comment:
```suggestion
            dname = key.replace(f'_{fname}','').replace('_var','')
```

Use "new" and better string syntax